src/java/htsjdk/tribble/bed/SimpleBEDFeature.java

```diff
@@ -136,4 +137,9 @@ public void setLink(String link) {
     public java.util.List<FullBEDFeature.Exon> getExons() {
         return emptyExonList;
     }
+
+    @Override
```

Comment:
Agreed.  My intuition is that some of the classes that now implement ```getInterval``` are not 1-based closed, and so those method need to be updated.